test/modules/iconSelect/components/IconSelect.test.js

```diff
@@ -161,6 +161,29 @@ describe('IconSelect', () => {
 	});
 
 	describe('when icon is selected (event handling) ', () => {
+		it('fires a "select" event', async () => {
+			spyOn(iconServiceMock, 'all').and.returnValue(Promise.resolve([new IconResult('foo', '42'), new IconResult('bar', '42')]));
+
+			const state = {
+				media: {
+					portrait: false
+				}
+			};
+			const element = await setup(state, {});
+			const spy = jasmine.createSpy();
+			element.addEventListener('select', spy);
+
+			element.click();
+			const selectableIcon = element.shadowRoot.querySelector('#svg_foo');
+			selectableIcon.click();
+
+			expect(spy).toHaveBeenCalledOnceWith(
+				new CustomEvent('toggle', {
+					detail: { selected: jasmine.any(IconResult) }
+				})
+			);
```

Comment:
I just stumbled over this naming issue and wonder why 
it doesn't matter if we test with 
`new CustomEvent('toggle', {detail: { selected: jasmine.any(IconResult) }})`
or 
`new CustomEvent('select', {detail: { selected: jasmine.any(IconResult) }})`
or 
`new CustomEvent('fooBar', {detail: { selected: jasmine.any(IconResult) }})`

All this variants fullfil the expectation.

We should think about a more precise expectation like 
`expect(spy).toHaveBeenCalledOnceWith(jasmine.objectContaining({ type: 'select', detail: { selected: jasmine.any(IconResult) } }));`
which fails, if the type is not 'select'.